test/e2e_test.go

```diff
@@ -94,6 +94,7 @@ var _ = Describe("MetricsServer", func() {
 		if err != nil {
 			panic(err)
 		}
+		time.Sleep(15 * time.Second)
```

Comment:
Please comment why this sleep is needed.

Reply:
In fact, the effective key point is in
```
Limits: map[corev1.ResourceName]resource.Quantity{
corev1.ResourceCPU: mustQuantity("100m"),
},
```
I will roll back this modification